sdk/python/kfp/dsl/_container_op.py

```diff
@@ -878,7 +879,16 @@ def add_pod_label(self, name: str, value: str):
         self.pod_labels[name] = value
         return self
 
-    def set_retry(self, num_retries: int, policy: str = None):
+    def set_priority_class_name(self, name: str):
+        """Adds a priority class to the pod
```

Comment:
Hi @munagekar , I recently formatted this file to use 2 spaces indentations. Can you please rebase your PR and make the indentation consistent within the file? Sorry about the hassle.

Once done rebasing, the PR can be merged.

Reply:
@chensun I have rebased (kind of 😅 ). PTAL.